web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx

```diff
@@ -36,15 +42,40 @@ function StorePlugin(props: StorePluginProps) {
     }
   }, [isInstallSuccess]);
 
+  const argsStoreError = [
+    <div
+      onMouseEnter={() => setShowTooltip(true)}
+      onMouseLeave={() => setShowTooltip(false)}
+    >
+      <FiAlertTriangle color="#FFA41D" />
```

Comment:
Do we have this color in the ui-kit? 🤔

Reply:
it is in react-icons/fi

Reply:
yes it is the same  color as warning

Reply:
https://github.com/massalabs/station/blob/591575d66641d91e41733b9bd6ea165bf5984bc1/web/massastation/src/pages/Store/StoreSection/StorePlugin.tsx#L50